### ISSUE TITLE
Textmate 2

### ISSUE THREAD

**Issue by route** [elixir-editors/elixir-tmbundle#114]
I'm on TM2 beta nightly builds and it looks like they gonna make it 2.0 soon, so currently they got rid of Avian directory so this won't work:
```
mkdir -p ~/Library/Application\ Support/Avian/Bundles
cd ~/Library/Application\ Support/Avian/Bundles
git clone git://github.com/elixir-lang/elixir-tmbundle Elixir.tmbundle
```

instead the directory was named as for TM1 version:

```
mkdir -p ~/Library/Application\ Support/TextMate/Bundles
cd ~/Library/Application\ Support/TextMate/Bundles
git clone git://github.com/elixir-lang/elixir-tmbundle Elixir.tmbundle
```
Maybe we should add an extra section for those who's on TM2.0 beta nightly builds... not sure

**Comment by josevalim** [comment]
Let's wait until the TM2 changes are officially released and then we can migrate. Wdyt?

**Comment by route** [comment]
Yep I just thought we could mention in README these lines about nightly builds and then eventually leave just them

**Comment by radiospiel** [comment]
I am on Textmate 2.0 rc4 now, and the dirs in the above example seem correct, as other bundles are also in that place. However, my Textmate fails to actually show the Elixir bundle anywhere in the UI, let alone let me use it for elixir source files. What am I missing here?

**Comment by brandondrew** [comment]
I just installed the bundle for TM2, using the TM1 directions.  All but the `osascript` line work fine.

**Comment by josevalim** [comment]
Can someone please confirm the instructions in the README are up to date and send a PR if not? Thank you.

**Comment by infininight** [comment]
The instructions for 2.0 work ok, for users there is a better location though:

    ~/Library/Application\ Support/TextMate/Pristine\ Copy/Bundles

This is the location used if TextMate installs a bundle via double-clicking the bundle. This location will never be written to for user changes making updating the bundle later easier. Rather than writing there a delta bundle will be created in the non-pristine folder with only the users changes.

I'll make that change in a PR I'm about to make.

**Comment by josevalim** [comment]
Being fixed as part of #155 .
